concordium-consensus/src/Concordium/ImportExport.hs

```diff
@@ -1,6 +1,7 @@
 {-# LANGUAGE ScopedTypeVariables #-}
 {-# LANGUAGE TemplateHaskell #-}
 {-# LANGUAGE TypeApplications #-}
+{-# OPTIONS_GHC -Wno-unused-matches #-}
```

Comment:
This is a bad idea. Please remove it. If there is an impossible case please add it explicitly and document why it is impossible, e.g., `error "Cannot happen because I say so."`.

Reply:
HLS likes to add compiler options without being asked to...